joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/passes/PhpTypeStubsParser.scala

```diff
@@ -27,14 +31,15 @@ case class KnownFunction(
   *
   * TODO: Need to handle variadic arguments.
   */
-class PhpSetKnownTypesPass(cpg: Cpg, knownTypesFile: Option[JFile] = None)
+class PhpTypeStubsParserPassGenerator(cpg: Cpg, config: XTypeStubsParserConfig = XTypeStubsParserConfig())
```

Comment:
The reason the `XTypeRecoveryPassGenerator` is named a generator is because it generates passes. This pass doesn't so it should stay `PhpSetKnownTypesPass`

Reply:
Makes sense, fixed.